SwiftUIBasics/Views/PlansView.swift

```diff
@@ -7,12 +7,92 @@
 
 import SwiftUI
 
-struct PlansView: View {
+struct ContentView2: View {
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack(alignment:.center){
+            Text("Choose")
+                .font(.system(size: 32, weight: .black, design: .serif))
+                .fontDesign(.rounded)
+            Text("Your Plan")
+                .font(.system(size: 32, weight: .black, design: .serif))
+                .fontDesign(.rounded)
+            HStack {
+                CardView(iconName: "", title: "Basic", description: "Per month", price: "$9")
```

Comment:
icon name could be set as default to be used like
```suggestion
                CardView(title: "Basic", description: "Per month", price: "$9")
```